### PR TITLE
Enable log/slog from Go v1.21 and migrate scheduler.

### PR DESCRIPTION
**Pull request by calebbrown**
Go 1.21 added `log/slog`. This PR wires up Zap logger to slog, so that logs written to `slog` are handled by Zap.

This change also adds an `slog.Handler` for adding `slog.Attr`s to a context.

### CODE REVIEW

File: sandboxes/staticanalysis/Dockerfile [hunk 1]
```diff
@@ -1,4 +1,4 @@
-FROM golang:1.19-alpine@sha256:122f3484f844467ebe0674cf57272e61981770eb0bc7d316d1f0be281a88229f as build
+FROM golang@sha256:12359f96c43077803959dab88ce60c87cd2543f3b907bf54a34a5104270f404f as build
```
Comment (maxfisher-g) [inline sandboxes/staticanalysis/Dockerfile:1]: nit: can we keep the build tag here too?

File: cmd/analyze/Dockerfile [hunk 2]
```diff
@@ -1,4 +1,4 @@
-FROM golang@sha256:122f3484f844467ebe0674cf57272e61981770eb0bc7d316d1f0be281a88229f as build
+FROM golang@sha256:12359f96c43077803959dab88ce60c87cd2543f3b907bf54a34a5104270f404f as build
```
Comment (maxfisher-g) [inline cmd/analyze/Dockerfile:1]: nit: can we add the build tag here (see static analysis dockerfile)

File: cmd/scheduler/Dockerfile [hunk 3]
```diff
@@ -1,4 +1,4 @@
-FROM golang@sha256:122f3484f844467ebe0674cf57272e61981770eb0bc7d316d1f0be281a88229f as build
+FROM golang@sha256:12359f96c43077803959dab88ce60c87cd2543f3b907bf54a34a5104270f404f as build
```
Comment (maxfisher-g) [inline cmd/scheduler/Dockerfile:1]: nit: can we add the build tag here (see static analysis dockerfile)

**Review by maxfisher-g — APPROVED** [review]
nice one, just some nits around build tags

File: sandboxes/staticanalysis/Dockerfile (hunk 1 above)
Comment (calebbrown) [reply sandboxes/staticanalysis/Dockerfile]: Done

File: cmd/scheduler/Dockerfile (hunk 3 above)
Comment (calebbrown) [reply cmd/scheduler/Dockerfile]: Done

File: cmd/analyze/Dockerfile (hunk 2 above)
Comment (calebbrown) [reply cmd/analyze/Dockerfile]: Done

**Comment by calebbrown** [comment]
See #185

**Pull request merged**